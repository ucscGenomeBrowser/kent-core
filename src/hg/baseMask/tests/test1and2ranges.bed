chr1	1	10
chr1	20	30
chr2	74582	79225
chr2	79225	144470
