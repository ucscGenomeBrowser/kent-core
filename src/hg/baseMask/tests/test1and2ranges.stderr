69907 bases in 4 ranges in 2 chroms in baseMask
